api-reference/50 Common/Object Structures/positionConfig/offset/offset.md

```diff
@@ -4,26 +4,113 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-Specifies horizontal and vertical offset in pixels.
+Specifies the offset of the overlay element from the specified position.
 
 ---
-This options accepts an object containing the **x** and **y** fields which specify the horizontal and vertical offset respectively, or a string consisting of horizontal and vertical offset values separated separated by a space (e.g., "5 -10").
+The offset is specified in pixels. To set this option, use an object with the **x** and **y** fields or a string value that indicates the horizontal and vertical offsets separated by a whitespace character. Positive offsets shift the element right or down, negative offsets shift it left or up. 
```

Comment:
Can we be more specific than "use an object with the **x** and **y** fields"
(See my similar suggestions above)

"or a string value that indicates the horizontal and vertical offsets separated by a whitespace character"
Change to "or a string value that indicates horizontal and vertical offsets separated by a whitespace character" (simply remove "the")

For the last sentence:
A positive offset shifts an element to the right or down; a negative offset shifts it to the left or up.